iroh-resolver/src/unixfs_builder.rs

```diff
@@ -638,6 +643,41 @@ mod tests {
         Ok(())
     }
 
+    async fn read_to_vec<T: AsyncRead + Unpin>(mut reader: T) -> Vec<u8> {
+        let mut out = Vec::new();
+        reader.read_to_end(&mut out).await.unwrap();
+        out
+    }
+
+    #[tokio::test]
+    async fn test_builder_roundtrip_huge() -> Result<()> {
```

Comment:
can we rename these from `_huge` and `_large` etc to `_128m` and the like. I have wildly different expectations from huge. My expectation is that large and huge are at least in the high GB or low TB range for a single request.

Reply:
Yes, originally was 1gb. I made it smaller so it runs faster.

So, what do we do with this test: it is not passing, yet it should be on main. So, should I just disable it.

By the way - I will do a lot WRT testing. For something that is just a data transformation, like going from a file to unixfs and back, there are not many excuses to have bugs.